### ISSUE TITLE
Run accessibility audit to find barriers that fail the Web Content Accessibility Guidelines 2.1 at level A and AA

### ISSUE THREAD

**Issue by NickColley** [nolanlawson/pinafore#2235]
We know of some issues but there's definitely more, we need to an audit.

**Comment by NickColley** [comment]
Going to run this in a separate repo to avoid blowing up this projects issues:

@nolanlawson this is the scope I'm thinking at the moment, anything high priority missed?

https://github.com/NickColley/pinafore-accessibility-review/issues/1
